tests/compiler/test_scheduling.py

```diff
@@ -0,0 +1,69 @@
+"""
+    Copyright 2021 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+
+from inmanta import compiler
+
+
+def test_function_rescheduling(snippetcompiler):
```

Comment:
```suggestion
def test_2787_function_rescheduling(snippetcompiler):
```

Reply:
great minds, ... (I had forgotten to push this change)